azure-quantum/tests/README.md

```diff
@@ -1,5 +1,19 @@
 # Unit tests
 
+The 'recordings' directory is used to replay network connections.
+To manually create new recordings, remove the 'recordings' subdirectory and run the tests.
```

Comment:
Thanks for adding these instructions! Tagging @ricardo-espinoza since he recently asked about it.